### PR TITLE
Fix typo in map.ts

### PR DESCRIPTION
**Pull request by eltociear**
happend -> happened

### CODE REVIEW

File: src/vs/base/common/map.ts [hunk 1]
```diff
@@ -887,7 +887,7 @@ export class LinkedMap<K, V> implements Map<K, V> {
 			this._tail = undefined;
 		}
 		else if (item === this._head) {
-			// This can only happend if size === 1 which is handle
+			// This can only happened if size === 1 which is handle
```
Comment (augustomaneiro) [inline src/vs/base/common/map.ts:890]: Shouldn't it be `This can only happen if size === 1 which is handled`?

File: src/vs/base/common/map.ts [hunk 2]
```diff
@@ -896,7 +896,7 @@ export class LinkedMap<K, V> implements Map<K, V> {
 			this._head = item.next;
 		}
 		else if (item === this._tail) {
-			// This can only happend if size === 1 which is handle
+			// This can only happened if size === 1 which is handle
```
Comment (augustomaneiro) [inline src/vs/base/common/map.ts:899]: Shouldn't it be `This can only happen if size === 1 which is handled`?

**Review by augustomaneiro — COMMENTED** [review]
Check comment

**Comment by eltociear** [comment]
@augustomaneiro Thank you for the information. I think you're right.

**Review by connor4312 — CHANGES_REQUESTED** [review]
See Augusto's comments

**Comment by eltociear** [comment]
@connor4312 I fixed it.
please re-review.

**Pull request merged**